examples/playground/index.html

```diff
@@ -201,5 +201,43 @@ <h5 style="margin-top: 0.1rem; margin-bottom: 0.2rem">Help</h5>
 
       <a-entity light="type: point; color: #f4f4f4; intensity: 0.4; distance: 0" position="0 2 0"></a-entity>
     </a-scene>
+
+    <script>
+      // Add the drag and drop event listeners to the document and process a dragged GLB file.
+      document.addEventListener(
+        "dragover",
+        function (e) {
+          e.stopPropagation();
+          e.preventDefault();
+          e.dataTransfer.dropEffect = "copy";
+        },
+        false
+      );
+
+      document.addEventListener(
+        "drop",
+        function (e) {
+          e.stopPropagation();
+          e.preventDefault();
+
+          const files = e.dataTransfer.files; // Array of all files.
+
+          for (const file of files) {
+            // Process only GLB files.
+            if (file.name.endsWith(".glb")) {
```

Comment:
Maybe we should recognize .bin files or leave a notification to people that their Hubs scene files may be named .bin but be glb format... I could submit a PR if helpful.

Reply:
It's not really standard to name a glb .bin. What is standard is a .gltf with a .bin, not .bin alone. I hope the official hubs export will name the file .glb. You also need to rename to .glb if you want to test it on https://gltf-viewer.donmccurdy.com/